### PR TITLE
HomeWork Exception

### PR DESCRIPTION
**Pull request by qpuhuk**
Hw is done

### CODE REVIEW

File: src/main/java/AlexLink/HomeWork/Exception/Car.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    void start() throws StartException {
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/Exception/Car.java]: you special use default access modifier or missed it?

Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/Exception/Car.java]: fixed it

File: src/main/java/AlexLink/HomeWork/Exception/StartException.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package AlexLink.HomeWork.Exception;
+
+public class StartException extends Exception {
```
Comment (atlamax) [inline src/main/java/AlexLink/HomeWork/Exception/StartException.java]: @svetamolchanova Is this expected that this class extends Exception, not RuntimeException?

Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/Exception/StartException.java]: fixed

**Pull request merged**